components/member-card/card.module.scss

```diff
@@ -79,6 +79,58 @@
   font-size: 0.88rem;
 }
 
+.walletContainer {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  align-items: center;
+  border-radius: 2px;
+  width: 150px;
+  height: 70px;
+  padding: 10px 5px 2px;
+  margin-bottom: 8px;
+  margin-left: 6px;
+  background-color: #EEEEEE;
+
+  .coinData {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .coinData>p {
+    height: 2px;
+  }
+
+  .coinData>p:first-child {
+    font-size: 12px;
+    font-weight: 500;
+    transform: translateY(-25px);
+    margin-right: 8px;
+  }
+
+  .coinData>p:last-child {
+    font-size: 20px;
+    font-weight: 700;
+  }
+
+  .coinDinero{
+    background-color: green;
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    box-shadow: rgb(0, 0, 0) 3px 3px 3px;
+    border: 1px solid green;
+  }
+  .coinNeelam{
+    background-color: blue;
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    box-shadow: rgb(0, 0, 0) 3px 3px 3px;
+    border: 1px solid blue;
+  }
```

Comment:
Almost both of them have the same CSS except for the background.
Why not change the className to just `coin` and directly use the background color (inline style)